src/xy/__tests__/xyEnsureFloat64.test.ts

```diff
@@ -1,4 +1,4 @@
-import { xyEnsureFloat64 } from '../xyEnsureFloat64';
+import { xyEnsureFloat64 } from '../index';
```

Comment:
why?

Reply:
it was a suggestion some time ago, to ensure that I am exporting the function.